projects/ad9361/src/main.c

```diff
@@ -598,14 +612,14 @@ int main(void)
 #endif
 	axi_dac_init(&ad9361_phy->tx_dac, &tx_dac_init);
 	axi_adc_init(&ad9361_phy->rx_adc, &rx_adc_init);
-	extern const uint32_t sine_lut_iq[1024];
+//	extern const uint32_t sine_lut_iq[1024];
 	axi_dac_set_datasel(ad9361_phy->tx_dac, -1, AXI_DAC_DATA_SEL_DMA);
-	axi_dac_load_custom_data(ad9361_phy->tx_dac, sine_lut_iq,
-				 NO_OS_ARRAY_SIZE(sine_lut_iq),
-				 (uintptr_t)dac_buffer);
-#ifdef XILINX_PLATFORM
-	Xil_DCacheFlush();
-#endif
+//	axi_dac_load_custom_data(ad9361_phy->tx_dac, sine_lut_iq,
+//				 ARRAY_SIZE(sine_lut_iq),
+//				 (uintptr_t)dac_buffer);
+//#ifdef XILINX_PLATFORM
+//	Xil_DCacheFlush();
+//#endif
```

Comment:
We shouldn't have commented code. If we don't need it, just delete it. This isn't a real rhyme, by the way, it's only `it` rhyming with itself. :)

Reply:
Modifications in main.c were provided for better showcasing the operation of the driver. These will be reverted.